### ISSUE TITLE
[A11y] Search Results - Grid - Contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1920]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

Two minor contrast issues:

- The current selection white on orange for both the "grid" icon and the "1" (page number) fail contrast
- The "Prev" link fails contrast, though just barely (the ratio is like 4.478:1 and needs to be 4.5:1)

### Suggested Fix

I think both of these exist in other pages, so if the code is shared, this issue is likely a dupe.  If not, same fix:

- Get rid of the orange / white combo or make the orange darker
- Make "Prev" a slightly darker gray

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

The Accessibility Insights tool for windows has a color picker.  Some automated tools fail to catch all contrast problems and some flag problems that aren't real, so be careful not to just use an automated tool.

**Comment by CGillen** [comment]
Outdated, fixed by #1987
